src/cluster_asm.c

```diff
@@ -305,17 +305,17 @@ sds asmCatInfoString(sds info) {
                         "cluster_slot_migration_active_trim_started:%llu\r\n"
                         "cluster_slot_migration_active_trim_done:%llu\r\n"
                         "cluster_slot_migration_active_trim_cancelled:%llu\r\n"
-                        "cluster_slot_migration_active_trim_keys_total:%llu\r\n"
-                        "cluster_slot_migration_active_trim_keys_deleted:%llu\r\n",
+                        "cluster_slot_migration_active_trim_current_job_keys:%llu\r\n"
+                        "cluster_slot_migration_active_trim_current_job_trimmed:%llu\r\n",
```

Comment:
These names became a bit long. Maybe we don't need to start with `cluster_`, not sure.